Source/Scene/QuadtreePrimitive.js

```diff
@@ -505,7 +505,8 @@ define([
     }
 
     function screenSpaceError(primitive, frameState, tile) {
-        if (frameState.mode === SceneMode.SCENE2D) {
+        // TODO: check for sseDenominator so shadow cameras with orthographic frustum works. check correctness.
```

Comment:
Definitely relevant for the shadows-globe branch, but I can remove it from this.